cpp-client/deephaven/client/include/private/deephaven/client/server/server.h

```diff
@@ -107,6 +110,7 @@ class Server : public std::enable_shared_from_this<Server> {
       std::unique_ptr<TableService::Stub> tableStub,
       std::unique_ptr<ConfigService::Stub> configStub,
       std::unique_ptr<arrow::flight::FlightClient> flightClient,
+      const deephaven::client::ClientOptions::extra_headers_t &extraHeaders,
```

Comment:
Shorten name and then pass by value (we will change server.cc to use std::move)
```
ClientOptions::extra_headers_t extraHeaders,
```

Reply:
Cool, I am liking this idiom, getting more assimilated in my brain as I use it more.